src/commands/edge/drivers/package.ts

```diff
@@ -63,15 +65,51 @@ export default class PackageCommand extends EdgeCommand {
 		upload: flags.string({
 			char: 'u',
 			description: 'upload zip file previously built with --build flag',
-			exclusive: ['build'],
+			exclusive: ['build-only'],
+		}),
+		assign: flags.boolean({
+			char: 'a',
+			description: 'ask user for a channel to assign the driver to after upload',
+			exclusive: ['assign-to', 'build-only'],
+		}),
+		'assign-to': flags.string({
+			description: 'automatically assign driver to specified channel after upload',
+			exclusive: ['assign', 'build-only'],
+		}),
+		install: flags.boolean({
+			char: 'I',
+			description: 'ask user for hub to install to after assigning it to the channel, implies --assign if --assign or --assign-to not included',
+			exclusive: ['install-to', 'build-only'],
+		}),
+		'install-to': flags.string({
+			description: 'automatically install driver to specified hub, implies --assign if --assign or --assign-to not included',
+			exclusive: ['install', 'build-only'],
```

Comment:
I suggest replacing `ask user` with `prompt` or just `ask`.

Reply:
I like `prompt` so I've updated this and `--assign` with that wording. I also fixed the wording in the examples below.